docs/topics/i18n/translation.txt

```diff
@@ -70,6 +70,20 @@ as a shorter alias, ``_``, to save typing.
        global ``_()`` function causes interference. Explicitly importing
        ``ugettext()`` as ``_()`` avoids this problem.
 
+    .. admonition:: What functions may be aliased as ``_``?
+
+        Only these functions can be imported as ``_``:
+
+        * :func:`~django.utils.translation.gettext`
+        * :func:`~django.utils.translation.gettext_lazy`
+        * :func:`~django.utils.translation.ugettext`
+        * :func:`~django.utils.translation.ugettext_lazy`
+
+        :djadmin:`makemessages` won't work as expected if any other
```

Comment:
In that spirit, is it redundant to say, "Only these functions" and then give examples of other functions? ;-)